packages/web-extension/src/walletManager/walletManager.types.ts

```diff
@@ -46,11 +53,28 @@ export interface WalletManagerActivateProps<P extends string | number = string,
 }
 
 export interface WalletManagerApi {
+  chainId$: Observable<Cardano.ChainId>;
```

Comment:
I think there's currently no way to know which wallet is active
```suggestion
  chainId$: Observable<Cardano.ChainId>;
  activeWalletId$: Observable<WalletId>;
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/813029a3123dac04ffb0f783c6e239a3c19ae8e9

Reply:
Let's combine `activeWalletId$` and `activeWalletAccount$` into 1 `activeWallet$: Observable<{walletId: WalletId; accountIndex?: number}>`, because they are always set together in 1 operation. Depending on decision https://github.com/input-output-hk/cardano-js-sdk/pull/1012#discussion_r1427616805 we could also add `chainId: ChainId` to it.

Reply:
changes as discussed in slack, done here: https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/cd24430f838afcaaa5d8271ec804d2f53a167ad4